### DISCUSSION TITLE
Add speed, power, heading topics to MQTT

### DISCUSSION THREAD

**Discussion by eliluong** [teslamate-org/teslamate#1347] [Ideas]
Hi, is it possible for us to have additional MQTT topics for speed, power, and heading? I believe these should be available from data from the websocket streaming API. Thanks.

**Comment by adriankumpf** [comment]
Check out https://docs.teslamate.org/docs/integrations/mqtt

Speed: `teslamate/cars/$car_id/speed`
Heading: `teslamate/cars/$car_id/heading`

Power is still missing. This would have to be added.

**Reply by eliluong** [reply]
Thanks, missed those two. Do you think the streaming API and MQTT update fast enough to have close to real-time numbers? I'm not sure how frequent it is updating.
